controllers/admin/AdminModulesController.php

```diff
@@ -124,10 +124,10 @@ public function __construct()
     public function checkCategoriesNames($a, $b)
     {
         if ($a['name'] === $this->trans('Other Modules')) {
-            return true;
+            return 0;
```

Comment:
Hi @Prestaworks If you want to fit the old logic you should use instead:
```php
return 1
```

Reply:
https://www.php.net/manual/en/function.uasort
actually if the values are equal, a zero is expected.
"The comparison function must return an integer less than, equal to, or greater than zero if the first argument is considered to be respectively less than, equal to, or greater than the second. "

Reply:
Yes you're right and here the exepected behavior is to consider than "Other Modules" should always be first, then you always have to return 1 otherwise it will be compared as equal than others.

Reply:
@Prestaworks 
Can you fix this line to `return 1;`?
If I understand well this is the only thing that is keeping this PR from being merged.